spec/views/idv/shared/_document_capture.html.erb_spec.rb

```diff
@@ -11,29 +11,42 @@
   let(:selfie_image_upload_url) { nil }
 
   before do
-    allow(view).to receive(:flow_session).and_return(flow_session)
-    allow(view).to receive(:sp_name).and_return(sp_name)
-    allow(view).to receive(:failure_to_proof_url).and_return(failure_to_proof_url)
-    allow(view).to receive(:front_image_upload_url).and_return(front_image_upload_url)
-    allow(view).to receive(:back_image_upload_url).and_return(back_image_upload_url)
-    allow(view).to receive(:selfie_image_upload_url).and_return(selfie_image_upload_url)
     allow(view).to receive(:url_for).and_return('https://example.com/')
+
+    allow(FeatureManagement).to receive(:document_capture_async_uploads_enabled?).
+      and_return(async_uploads_enabled)
+
+    assign(:step_url, :idv_doc_auth_step_url)
```

Comment:
this is the key change in this file, now that the partial uses `@step_url`. The rest of the changes in the file were to simplify and minimize stubbing